performance_regression_lwt_test.py

```diff
@@ -0,0 +1,36 @@
+from invoke.exceptions import UnexpectedExit, Failure
+
+from performance_regression_test import PerformanceRegressionTest
+from sdcm.utils.common import log_run_info, retrying
+
+
+class PerformanceRegressionLWTTest(PerformanceRegressionTest):
+
+    """
+    Test Scylla performance regression with cassandra-stress.
+    """
+
+    @log_run_info
+    @retrying(n=3, sleep_time=15, allowed_exceptions=(UnexpectedExit, Failure))  # retrying since SSH can fail with 255
```

Comment:
why does it fail?

Reply:
Dunno, it is copy paste from other tests